jobs/ci-kubernetes-kubemark-5-gce-1.5.env

```diff
@@ -4,7 +4,7 @@ ENABLE_GARBAGE_COLLECTOR=true
 PROJECT=k8s-kubemark-1-5
 USE_KUBEMARK=true
 KUBEMARK_TESTS=\[Feature:Empty\]
-KUBEMARK_TEST_ARGS=--gather-resource-usage=true --garbage-collector-enabled=true
+KUBEMARK_TEST_ARGS=--gather-resource-usage=true --garbage-collector-enabled=true --output-print-type=json
```

Comment:
Since you're touching this, can you remove "--garbage-collect-enabled=true"?
This is default.

Same in other places.